packages/google_sign_in/example/lib/main.dart

```diff
@@ -50,7 +50,8 @@ class SignInDemoState extends State<SignInDemo> {
       _contactText = "Loading contact info...";
     });
     http.Response response = await http.get(
-        'https://people.googleapis.com/v1/people/me/connections',
+        'https://people.googleapis.com/v1/people/me/connections' +
+            '?requestMask.includeField=person.names%2Cperson.email_addresses',
```

Comment:
Do we actually need the email addresses? From the code it looks like names should be enough.